quickannotator/db/models.py

```diff
@@ -83,11 +84,11 @@ class Tile(Base):
     tile_id = Column(Integer, nullable=False)
 
     # columns
-    pred_status = Column(Integer, nullable=True, default=TileStatus.UNSEEN)
-    pred_datetime = Column(DateTime, nullable=True, server_default=func.now())
+    pred_status = Column(Integer, nullable=False, default=TileStatus.UNSEEN)
```

Comment:
pred_status should not be nullable since the default state is `UNSEEN`. Do you agree?

Reply:
UhhhhhhhHHHHhhhh...hmm....yes? yes. i think so. so this is basically to say that there is no "less active" state than UNSEEN? yea, it checks out